code/src/index.css

```diff
@@ -1,13 +1,201 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: "Helvetica", "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", 
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: black;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: 'Helvetica', sans-serif;
+}
+
+h1 {
+  color: #FFFFFF;
+  letter-spacing: 1.7px;
+  font-size: 1.25rem;
+}
+
+h2 {
+  color: #FFFFFF;
+  text-transform: uppercase;
+  padding: 0.5em 0.75em;
+  margin-top: 0.5em;
+  border-bottom: 1px solid #a0a0a070;
+  border-top: 3px solid #a0a0a070;
+}
+
+h3 {
+  color: #FFFFFF;
+  font-size: 1.125rem;
+  letter-spacing: 1.6px;
+  max-width: 300px;
+  margin: 0.35em 0em 0.5em;
+}
+
+a.artist {
+  color: #a0a0a0;
+  margin: 0em 0em 1.5em;
+}
+
+/***** Header *****/
+header {
+  margin: 1em 0.5em;
+}
+
+/***** Links *****/
+.nav-link,
+.nav-link:active {
+  color: #FFFFFF;
+  text-decoration: none;
+}
+
+.nav-link:hover,
+.nav-link:focus {
+  text-decoration: underline;
+}
+
+/***** Icons *****/
+.icon {
+  width: 35px;
+  filter: brightness(0) invert(0.9);
+}
+
+.icon.play {
+  width: 60px;
+}
+
+/***** Layout *****/
+.container {
+  padding: 1em 2em;
+}
+
+.album-container,
+.single-container,
+.playlist-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  justify-items: center;
+  grid-gap: 1em;
+  grid-row-gap: 2em;
+}
+
+.artist-container {
+  display: flex;
+}
+
+.artist-container a:not(:first-of-type, :last-of-type):before {
+  content: ', ';
+} 
+
+.artist-container a:nth-last-of-type(2):after {
+  content: ' & ';
+  padding-right: 0.3em;
+}
+
+/***** Overlay *****/
+.album-group {
+  position: relative;
+}
+
+.album {
+  display: block;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  opacity: 0;
+  transition: 0.5s ease;
+}
+
+.album-group:hover .icon-container {
+  opacity: 1;
+  cursor: pointer;
+}
+
+.album-group:hover .album{
+  opacity: 0.3;
+  cursor: pointer;
+}
+
+.icon-container {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: 50%;
+  bottom: 15%;
```

Comment:
We think that this could be centered by making the overlay class to act like flexbox.